### ISSUE TITLE
Need new quick-protobuf release for pb-rs 0.9.0

### ISSUE THREAD

**Issue by therealprof** [tafia/quick-protobuf#165]
Rust code generated by pb-rs 0.9.0 (due to #158) requires the new implementation in the quick-protobuf crate and hence a new release, too:
```
error[E0432]: unresolved import `quick_protobuf::WriterBackend`
  --> src/usp.rs:15:70
   |
15 | use quick_protobuf::{MessageRead, MessageWrite, BytesReader, Writer, WriterBackend, Result};
   |                                                                      ^^^^^^^^^^^^^ no `WriterBackend` in the root
```

**Comment by tafia** [comment]
I'll do it tonight!

**Comment by tafia** [comment]
Published v0.7.0 for which I've also removed failure dependency.
